build.gradle

```diff
@@ -46,6 +46,19 @@ if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile
 	targetCompatibility = 22
 }
 
+if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile == "jre21")) {
```

Comment:
```suggestion
if (hasProperty('buildProfile') && buildProfile == "jre21") {
```

We want default to be 22.

Reply:
Never mind, if there is no build profile it will build with 22, no need to remove.